### DISCUSSION TITLE
Formulate working vision, goals, and objectives

### DISCUSSION THREAD

**Discussion by tonyfast** [materialsgenomefoundation/strategic-plan#5] [Ideas]
![image](https://github.com/materialsgenomefoundation/strategic-plan/assets/4236275/c9e954f9-a209-4158-81a5-24765dfcb1b5)

i've been working hard about to present a story that complements as much on going work as possible. in this figure, i illustrate some of my thoughts. a lot of our vision is influenced by where folks are shining the light. MDF, Marda have infrastructure. NIST, NASA have policy. the big centers have funding; the have people, space, and cash. i feel like our strength may be to shine light elsewhere and consider what happens.

the currently state of MGI funding really seems to favor centers that are resourced well enough to concurrently work on policy, funding, and infrastructure. these experiences isolate resources, and excludes under-resourced organizations from participating equitably in materials science. there still seems to be an intense nostalgia for face to face meetings which also serves resourced people. not participating in every primary aspect puts researchers at a disadvantage. 

we are missing things in the dark when we talk about MGI. the MGI is started and now we've created new problems for ourselves. every modern problem in materials science requires making the field and opportunities to participate more equitable. measurement equipment has become cheaper, software is free, and data is available. we need to start working under these conditions.  materials science is so general that its power shouldn't be so isolated and inaccessible.

the MGI successes generated new open science challenges that depend on a community, events, and software needs for the future workforce. open communities have the ability to self govern (to extent) that may find alternative policies and funding. open software is a mission we embody as a core value. our digital, jupyterhub interactive events, bolstered by our community, will be exceptional relative to what others are doing. 

between our community and events, we can build new experiences, and opportunities for others to host their own. community and software converge on code sharing platforms that effectively unify folks by their provider of Github, Gitlab, Gitea. this culture shift opens the opportunity for international collaborations that might have been thwarted by policy shortcomings. events and software introduce new opportunities for nimble funding to support open science and software.

through this thinking. i see three goals for the MGF

1. Grow online and in-person communities of materials scientists across generations, disciplines and expertise.
2. Host immersive open materials science training events, workshops, and conferences.
3. Develop sustainable open source software projects that increase access to materials data, theory, and models.

i've been wondering if community should consume the events, but i think they are different problems. supporting community on the daily is different than throwing a party.

**Comment by richardotis** [comment]
Really liking your thinking and the visual metaphor presented for this. Also, I agree that community development stands on its own from events. There is overlap between events and community, but I'd argue there is overlap between events and software, and community and software, as well (events help us onboard new users/contributors to the software in our portfolio, and open source software projects are natural substrates on which communities can form). It makes sense that the goals are complementary to a degree.

**Comment by tonyfast** [comment]
thanks for the feedback @richardotis . i'm moving this issue to a discussion so we can keep working on the vision and goals in the open there. i've opened a pull request to add these draft ideas to the existing document in #4

**Comment by richardotis** [comment]
I've been thinking it through some more, and now I'm wavering on whether events should be a top-level goal or not. What might the third top-level goal be, if not that?

**Comment by tonyfast** [comment]
i agree. i think events can ball into community. i think we are missing our focus or careers and jobs. that has been a theme, but it doesn't fit well into the way we've laid things out.

**Reply by richardotis** [reply]
My thought on top-level goal versus sub-goal is this: A top-level goal is something we seek to do because we view it as a good in-and-of-itself; we don't have to justify that goal in terms of any other goal. I see the community and software goals in that way already. What other goals might relate to the core values (that we haven't explicitly defined so far) of the Foundation?

**Comment by richardotis** [comment]
Also, while I don't think DEIA needs to be a top-level goal (it fits into the community goal nicely), I'm thinking that a sub-goal might involve increasing access to materials education and career opportunities for under-represented groups. I'm particularly thinking that our competencies in virtual/remote would align well with an effort to increase the number of materials scientists and engineers who identify as disabled. (BLS numbers I've seen on disabled representation do not reflect well on the engineering profession)

**Reply by tonyfast** [reply]
is increasing opportunity a goal? right now the opportunities to meet with community is a hurdle. could initialize goals be to increase opportunity to participate or identify with the materials science community?

**Comment by tonyfast** [comment]
how about something like?

Provide  __services__ that educate, connect, and train open materials scientists.

**Reply by richardotis** [reply]
Along these lines, the first goal is about community, I'm wondering if the third goal shouldn't be about individuals.

**Comment by tonyfast** [comment]
![image](https://github.com/materialsgenomefoundation/strategic-plan/assets/4236275/355e8e50-7a32-4839-b89c-c48a73d31f48)

i'm sharing an updated diagram about the MGF vision with notes following:

* MGF builds on the pillars of community, services, and software that support open materials research
* MGF complements the MGI efforts by providing infrastructure to build open materials research tools.
* the MGF moves money, people, and resources into the larger Materials Genome effort. these components primarily support the infrastructure needed to support community services that bolster the development of open materials tools.
* in this design, services generate new skills in materials software based on the novel, open cirricula that increase access to materials knowledge
* community growth brings new uses to materials science as the open software community continually provide new tools.
* we differentiate ourselves by bringing a radical open ethos to materials science to accommodate an increasingly complex technical landscape
